src/models/Resources.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const resourceSchema = new mongoose.Schema({
+    title: {
+        type: String,
+        required: [true, 'Cannot be blank'],
+        maxlength: 250,
+    },
+    topic: {
+        type: String,
+        required: [true, 'Cannot be blank'],
+        maxlength: 75,
+    },
+    subtopic: {
+        type: String,
+        required: [true, 'Cannot be blank'],
+        maxlength: 150,
+    },
+    type: {
+        type: String,
+        required: [true, 'Cannot be blank'],
+        enum :['Article', 'Video', 'Game'],
+        maxlength: 75,
+
+    },
+    link: {
+        type: String,
+        required: [true, 'Cannot be blank']
+    }
```

Comment:
add a createdBy field here so the front end team can retrieve the userId of the person who created it